src/server/components/lists/helpers.ts

```diff
@@ -30,6 +33,9 @@ export function preProcessParams(params: { [name: string]: any }): {
   const hasSelectedAll = paramsCopy?.practiceArea?.includes("All") ?? false;
   paramsCopy = hasSelectedAll === true ? { ...paramsCopy, practiceArea: "All" } : params;
   paramsCopy = paramsCopy?.region === "" ? { ...paramsCopy, region: "Not set"} : paramsCopy;
+  if (paramsCopy._csrf !== undefined) {
+    delete paramsCopy._csrf;
```

Comment:
I think you can just delete on keys that don't exist anyway (so no need to wrap in !== undefined check)